### ISSUE TITLE
Poly Line Crashes

### ISSUE THREAD

**Issue by bcoombs123** [smeighan/xLights#723]
I started using the Poly Line feature and every time I use it, it crashes xlights. The first two times I changed from RGB to one color and tried to change the color. It crashed! I had to restore the directory because it corrupted one of the files. The next time I was able to create one poly line and save. The second poly line I created, crashed when I changed the number of nodes from 50 to 300. Both times reports where sent to xlights from my PC.

**Comment by AzGilrock** [comment]
I see the crash reports (10/19 as a reminder for me).  Not sure why these steps are crashing for you.  A lot of people are using the model and doing the same things.  I'll be out of town so I can't look into this for at least 5 days.

**Comment by dkulp** [comment]
I tried reproducing it last night and did discover a crash if the starting location is set to Right, but it's a single channel type.   I will get that fixed, but setting it to the left (the default) should be OK.   Other than that, I couldn't reproduce.

**Comment by AzGilrock** [comment]
Yeah the whole model has issues when the location is set to Right.  I've worked on that but didn't get it right.  I was trying to do it like we do in Single Line Model where we just flip all the indices but it doesn't work for PolyLine.  It appeared to work when I first tested it because effects placed at the model level will work but once you place them at the strand level to target a line then they aren't working right.  I thought I had it fixed but when I made it work right for effects at the strand level then the model level effects were broken.
